### PR TITLE
blog post for argus

### PR DESCRIPTION
**Pull request by abhishekBansal**

### CODE REVIEW

**Comment by anujmiddha** [comment]
@aksswami please review and merge

File: _posts/2017-06-26-introducing-argus.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+With in-built support for Google and Facebook login / signup, Argus makes it trivial to get up and running when starting with a new app.
+
+A few lines of code can get you a screen looking like,
+
+![screenshot]({{ site.url }}/assets/images/argus_sample.png){:style="width: 250px; margin:auto;"}
+
+#### The motivation behind Argus
+
+At [moldedbits](http:///www.moldedbits.com), we do a lot of client projects. Be it an enterprise app, a social networking app or an app for a small business, most all apps need a user on-boarding flow, including login, registration forgot password. Having implemented almost similar flows across apps, we decided to abstract out the complexity into a separate library that could be reused.
```
Comment (aksswami) [inline _posts/2017-06-26-introducing-argus.md]: most all apps?

File: _posts/2017-06-26-introducing-argus.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Integrating in your apps
+
+We built `Argus` android to simplify our lives and to speed up app development. Argus is a library that makes user on-boarding hassle free.
+
+Argus works with providers for features like Login/Signup/ForgotPassword. These `Providers` take care of all flow and functionality of the feature they are responsible for. `Argus` comes with built-in providers for Google, Facebook and Email login/signup. These are sufficient for most of the use cases, however, if they aren't enough for you, you can create your own custom provider.
+
+Argus is fully customizable and it adapts look and feel of your application. Its open source and free to use.
```
Comment (aksswami) [inline _posts/2017-06-26-introducing-argus.md]: We can remove *It's open source and free to use. As we already have license in place for this

**Review by aksswami — CHANGES_REQUESTED** [review]

File: _posts/2017-06-26-introducing-argus.md (hunk 2 above)
Comment (aksswami) [reply _posts/2017-06-26-introducing-argus.md]: should remove.

**Review by aksswami — APPROVED** [review]

**Pull request merged**